knimin/lib/data_access.py

```diff
@@ -3191,6 +3186,252 @@ def delete_sample_plate(self, id):
             TRN.add(sql, [id])
             TRN.execute()
 
+    def get_id_by_name(self, field, name):
```

Comment:
I'm a bit confused - I thought this was adding only the interface? but there is a fair amount of python code in here for interacting with the DB.